css/main-header.styl

```diff
@@ -117,20 +116,18 @@
     margin: 0
 
 .main-nav-item
+  color: rgba(255,255,255,0.5) !important //ugh
```

Comment:
Ehhh, let's fix the original problem (`.main-header a`, right?) instead of this.